apps/projects/templates/a4_candy_projects/project_detail.html

```diff
@@ -287,18 +287,17 @@ <h1 class="project-header__title">{{ module.name }}</h1>
                     <div class="row">
                         <div class="col-md-10 col-lg-8 offset-md-1 offset-lg-2">
                           <h5>{% translate 'Insights' %}</h5>
-                        </div>
+                          <p class="form-hint"> {{ insight_label }} </p>
 
-                        <div class="col-md-10 col-lg-8 offset-md-1 offset-lg-2">
-                          <p> {{ insight_label }} </p>
-                        </div>
-                        <p>
-                        {% for label, value in counts %}
-                            <div class="col-md-10 col-lg-8 offset-md-1 offset-lg-2">
-                                {{ label|title }}: {{ value }}
+                          <dl class="d-flex flex-wrap">
```

Comment:
* this and the form need fixing on mobile
* the margin between the count and the label is much smaller in the design

Reply:
@philli-m also in the dashboard template, the design is plain, a list of labels and keys as it was before.

Reply:
oupps was using outdated design!

Reply:
@goapunk @m4ra should be fixed now

Reply:
@goapunk I don't follow your comments sorry, there's no margin between those elements? also I don't see an obvious breaking on mobile?